### PR TITLE
feat: Titus' neovim config prompt

### PR DESCRIPTION
**Pull request by pik4li**
## Type of Change
- [x] New feature

## Description
Made the setup of Chris's dotfiles as an option, so that you can only install neovim and its dependencies

## Testing
Changed the code -> save -> cargo run -> works

## Impact
small

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: core/tabs/applications-setup/Developer-tools/neovim-setup.sh [hunk 1]
```diff
@@ -51,9 +51,22 @@ linkNeovimConfig() {
...
+    N | n)
+        printf "%b\n" "${YELLOW}Installation complete.."
+        ;;
+    *)
+        cloneNeovim
+        backupNeovimConfig
+        linkNeovimConfig
+        ;;
+    esac
+}
```
Comment (adamperkowski) [inline core/tabs/applications-setup/Developer-tools/neovim-setup.sh:67]:
```suggestion
dotsPrompt() {
    printf "%b\n" "${GREEN}Do you want to install Titus' Neovim? (Y/n): ${RC}"
    read -r _titusDots
    case "$_titusDots" in
    N | n)
        printf "%b\n" "${YELLOW}Installation complete.."
        ;;
    *)
        cloneNeovim
        backupNeovimConfig
        linkNeovimConfig
        ;;
    esac
}
```

File: core/tabs/applications-setup/Developer-tools/neovim-setup.sh [hunk 2]
```diff
@@ -51,9 +51,22 @@ linkNeovimConfig() {
...
+
 checkEnv
 checkEscalationTool
 installNeovim
-cloneNeovim
-backupNeovimConfig
-linkNeovimConfig
+askToClone
```
Comment (adamperkowski) [inline core/tabs/applications-setup/Developer-tools/neovim-setup.sh:72]:
```suggestion
dotsPrompt
```

**Review by adamperkowski — CHANGES_REQUESTED** [review]

**Comment by adamperkowski** [comment]
I see where you're coming from but I'm not sure if we want that change. It may be better to just rename this from Neovim setup to something like Titus' Neovim setup

**Comment by pik4li** [comment]
> I see where you're coming from but I'm not sure if we want this change. It may be better to just rename the entry from Neovim setup to something like Titus' Neovim setup

Thanks very much. Im pretty new to contributing, so any help is appreciated

**Comment by ChrisTitusTech** [comment]
Extra keypresses and prompts just annoy me. Going to deny this PR, if you are rolling your own neovim... why would you use mine?

**Pull request closed without merging**